src/utils/rtl.tsx

```diff
@@ -1,35 +1,10 @@
 import { withRouter as _withRouter } from "@homebound/rtl-react-router-utils";
-import {
-  blur as _blur,
-  change as _change,
-  click as _click,
-  clickAndWait as _clickAndWait,
-  focus as _focus,
-  input as _input,
-  type as _type,
-  typeAndWait as _typeAndWait,
-  wait as _wait,
-  allowAndWaitForAsyncBehavior,
-  RenderResult,
-  render as rtlRender,
-  Wrapper,
-} from "@homebound/rtl-utils";
-import { prettyDOM } from "@testing-library/react";
+import { RenderResult, render as rtlRender, Wrapper } from "@homebound/rtl-utils";
 import { ReactElement } from "react";
-import { BeamProvider } from "src/components";
-import { fail } from "src/utils/index";
-export {
```

Comment:
I split up "the rtl.tsx that Beam's own tests use" (this file) and "rtlUtils.tsx the utilities that we bundle for downstream projects to use" (the next file).

Previously they were combined and should not have been.